app/templates/explore.hbs

```diff
@@ -1,6 +1,6 @@
 <div class="ui stackable grid">
   <div class="four wide column">
-    {{explore/side-bar model=model category=category sub_category=sub_category event_type=event_type date_range=date_range}}
+    {{explore/side-bar model=model category=category sub_category=sub_category event_type=event_type start_date=start_date end_date=end_date}}
```

Comment:
use camelCase

```hbs
{{explore/side-bar model=model category=category subCategory=sub_category eventType=event_type startDate=start_date endDate=end_date}}
```

Reply:
Should I change it everywhere?

Reply:
I mean even the query params?

Reply:
@niranjan94 These are not in camel case because the template is already using the camel-case version for iterating - 
```javascript
 {{#if (eq category Category.name)}}
          {{#each Category.subTopics as |subCategory|}}
            <a href="#"
               class="link inner item {{if (eq sub_category subCategory.slug) 'active'}}"
              {{action 'selectCategory' Category.name subCategory.slug}}>
              {{subCategory.name}}
            </a>
          {{/each}}
        {{/if}}
```

Reply:
All query params - snake_case
Every other js variable - camelCase

Reply:
But what about the camel cased variables(same name) in the template?

Reply:
Which template ? 
- If it is on the route's template, and it refers to the query param variable snake_case if fine since its the same variable.
- Anywhere else, camelCase

Reply:
The `explore/side-bar` template, the one which contains all the filtering options(FE wise). We are passing the values of the query params to it in the highlighted line. The problem is that the name of the query params is same as the model property if written in camel case. That's why we are passing variables in snake_case only. I have already pointed out an example(variable `sub_category` and `subCategory`) I will have to rename that `subCategory` to something else maybe `subCategoryItem`. Is that okay?

Variables which have this problem -> `sub_category` and `event_type`. Rest can be converted to camel case without a problem.